Added user secrets id in WebApi project file

ThorstenSauter opened the pull request:
Resolves #41

ThorstenSauter committed:
Added user secrets id in WebApi project file

pobiega reviewed (commented):
We need to include information/links on how to use user secrets for VS/Rider/dotnet CLI. Perhaps update the readme?

ThorstenSauter commented:
> We need to include information/links on how to use user secrets for VS/Rider/dotnet CLI. Perhaps update the readme?

Will providing a link to the [Microsoft documentation](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows#secret-manager) suffice? It mentions how to use it from the CLI, as well as VS.

For Rider users maybe we should include either an image or a screenshot like this one?
![image](https://user-images.githubusercontent.com/43311536/136241023-32838da6-250b-480a-8c62-a5fef214625e.png)

pobiega commented:
Yeah that seems okay to me.

ThorstenSauter committed:
Added documentation for user secrets in README.md

- Added documentation and an example for using user secrets
- Fixed some spelling mistakes and formatting
